### ISSUE TITLE
Unsupported keras layer type Conv1D

### ISSUE THREAD

**Issue by rockbass2560** [deeplearning4j/deeplearning4j#4678]
#### Issue Description

I got an error when I try to load a Keras model in dl4j

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version 0.9.1
* platform information (OS, etc)  Windows 10

#### Contributing

Log error

org.deeplearning4j.nn.modelimport.keras.UnsupportedKerasConfigurationException: Unsupported keras layer type Conv1D. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getKerasLayerFromConfig(KerasLayer.java:254)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperPrepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:99)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:56)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel$ModelBuilder.buildSequential(KerasModel.java:581)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialConfiguration(KerasModelImport.java:304)
	at Main.main(Main.java:18)

# Configuration of CNN in Keras

model = Sequential()
model.add(Conv1D(64,3, activation="relu", input_shape=(X_train.shape[1],X_train.shape[2])))
model.add(MaxPool1D(3))
model.add(Conv1D(64,3, activation="relu"))
model.add(MaxPool1D(3))
model.add(Conv1D(64,3, activation="relu"))
model.add(MaxPool1D(3))
model.add(Flatten())
model.add(Dense(500, activation="relu"))
model.add(Dropout(rate=0.5))
model.add(Dense(500, activation="relu"))
model.add(Dropout(rate=0.5))
model.add(Dense(15, activation="softmax"))
optim = Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=1e-08, decay=0.0)
model.compile(loss='sparse_categorical_crossentropy', optimizer=optim, metrics=['accuracy'])

model.fit(X_train,y_train,epochs=100,validation_data=(X_test, y_test))

**Comment by maxpumperla** [comment]
please use 0.9.2-SNAPSHOT

**Comment by rockbass2560** [comment]
How can I get that version?

**Comment by raver119** [comment]
https://deeplearning4j.org/snapshots
